docs/user_guides/integrations/databricks/api_key.md

```diff
@@ -15,127 +15,20 @@ For instructions on how to generate an API key follow this [user guide](../../pr
-Then click on the secret in the secrets list and take note of the *Secret ARN*.
-
-<p align="center">
-  <figure>
-    <img src="../../../../assets/images/guides/integrations/databricks/aws/databricks_secrets_manager_step_2.png" alt="Storing a Feature Store API key in the Secrets Manager Step 2">
-    <figcaption>Storing a Feature Store API key in the Secrets Manager Step 2</figcaption>
-  </figure>
-</p>
-
-Once the API Key is stored, you need to grant access to it from the AWS role that you have created in [Step 1](#step-1-create-an-instance-profile-to-attach-to-your-databricks-clusters).
-In the AWS Management Console, go to *IAM*, select *Roles* and then the role that that you have created in [Step 1](#step-1-create-an-instance-profile-to-attach-to-your-databricks-clusters).
-Select *Add inline policy*. Choose *Secrets Manager* as service, expand the *Read* access level and check *GetSecretValue*.
-Expand Resources and select *Add ARN*. Paste the ARN of the secret created in the previous step.
-Click on *Review*, give the policy a name and click on *Create policy*.
-
-<p align="center">
-  <figure>
-    <img src="../../../../assets/images/guides/integrations/databricks/aws/databricks_secrets_manager_policy.png" alt="Configuring the access policy for the Secrets Manager">
-    <figcaption>Configuring the access policy for the Secrets Manager</figcaption>
-  </figure>
-</p>
-
-#### Step 3: Allow Databricks to use the AWS role created in Step 1
-
-First you need to get the AWS role used by Databricks for deployments as described in [this step](https://docs.databricks.com/administration-guide/cloud-configurations/aws/instance-profiles.html#step-3-note-the-iam-role-used-to-create-the-databricks-deployment). Once you get the role name, go to *AWS IAM*, search for the role, and click on it. Then, select the *Permissions* tab, click on *Add inline policy*, select the *JSON* tab, and paste the following snippet. Replace *[ACCOUNT_ID]* with your AWS account id, and *[MY_DATABRICKS_ROLE]* with the AWS role name created in [Step 1](#step-1-create-an-instance-profile-to-attach-to-your-databricks-clusters).
-
-```json
-{
-    "Version": "2012-10-17",
-    "Statement": [
-        {
-            "Sid": "PassRole",
-            "Effect": "Allow",
-            "Action": "iam:PassRole",
-            "Resource": "arn:aws:iam::[ACCOUNT_ID]:role/[MY_DATABRICKS_ROLE]"
-        }
-    ]
-}
+```python hl_lines="6"
+    import hopsworks 
+    project = hopsworks.login(
+        host='my_instance',                 # DNS of your Feature Store instance
+        port=443,                           # Port to reach your Hopsworks instance, defaults to 443
+        project='my_project',               # Name of your Hopsworks Feature Store project
+        api_key_value='apikey',             # The API key to authenticate with Hopsworks
+        hostname_verification=True          # Disable for self-signed certificates
```

Comment:
The comment isn't consistent with the parameter.

 hostname_verification=False          # Disable hostname verification for self-signed certificates